src/index.html

```diff
@@ -11,7 +11,197 @@
+    method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form-field surname">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            required
+            autocomplete="off"
+            id="surname"
+          >
+        </div>
+
+        <div class="form-field name">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            id="name"
+          >
+        </div>
+
+        <div class="form-field age">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            id="age"
+          >
+        </div>
+
+        <div class="form-field birthday">
+          <label for="birthday">Full date of birth:</label>
+          <input
+            type="date"
+            name="birthday"
+            id="birthday"
+          >
+        </div>
+
+        <div class="form-field term_and_agreement">
+          <label for="term-and-agreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="term-and-agreement"
+            id="term-and-agreement"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="form-field email">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+            id="email"
+          >
+        </div>
+
+        <div class="form-field password">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="6"
+            maxlength="30"
+            id="password"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field cat">
+          <label for="cat">Do you love cats?</label>
+          <label>
+            <input
+              type="radio"
+              name="cat"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cat"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="form-field color">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </div>
+
+        <div class="form-field time">
```

Comment:
Remove additional calss names. They are useless
```suggestion
        <div class="form-field">
          <label for="password">Password:</label>
          <input
            type="password"
            name="password"
            required
            minlength="6"
            maxlength="30"
            id="password"
          >
        </div>
      </fieldset>

      <fieldset class="fieldset">
        <legend>An interesting fact about you!</legend>

        <div class="form-field">
          <label for="cat">Do you love cats?</label>
          <label>
            <input
              type="radio"
              name="cat"
              value="yes"
            >
            Yes
          </label>

          <label>
            <input
              type="radio"
              name="cat"
              value="no"
            >
            No
          </label>
        </div>

        <div class="form-field">
          <label for="color">What is your favorite color?</label>
          <input
            type="color"
            name="color"
            id="color"
          >
        </div>

        <div class="form-field">
```